pkgs/development/interpreters/ruby/gemconfig/default.nix

```diff
@@ -42,10 +42,16 @@ in
   };
 
   libv8 = attrs: {
-    buildInputs = [ which v8 python ];
+    dontBuild = false;
+    buildInputs = [ which v8 python git ];
     buildFlags = [
       "--with-system-v8=true"
     ];
+    # due to `git ls-files` ...
```

Comment:
I think this wouldn't happen if the gemspec comes from the metadata. Not 100% sure but I think the metadata contains the full file list.